daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ExprParser.scala

```diff
@@ -44,7 +44,7 @@ private[parser] class ExprParser[P](parserParameters: ParserParameters[P]) {
       (id ^? builtinFunctions) ^^ EBuiltin |
       caseOf |
       id ^^ EVar |
-      `(` ~> expr <~ `)`
+      (`(` ~> expr <~ `)`)
```

Comment:
This is to work around https://github.com/scala/bug/issues/12071. Afaict, the warning is a false positive but adding a few parentheses seems easy enough.